docs/UserGuide.md

```diff
@@ -2,16 +2,16 @@
 layout: page
 title: User Guide
 ---
-**Foster Family** is a desktop app built for the **foster managers of cat and dog shelters** to help them perform **foster family management tasks**. 
+**Foster Family** is a desktop application built for **foster managers of cat and dog shelters** to help the **administrative management** of **foster families**. 
 
 Here are some tasks which Foster Family can help you with: 
-* **Store and update** the important details of animal fosterers (people who temporarily care for an animal in their own homes)
-* Search for a fosterer using _**any**_ detail you can remember of them
-* **Gain insights** on the current status of fosterers
+* **Store and update** the important details of animal fosterers (people who temporarily care for an animal in their own homes).
+* **Search** for a fosterer using _any_ detail you can remember of them.
+* **Gain insights** on the overall status of managed fosterers.
 
-Foster Family is **optimised for use** via a Command Line Interface (CLI). This means you interact with it by typing commands in the command box. It also retains the benefits of a Graphical User Interface (GUI), allowing you to interact with the application through graphical components. If you can type fast, Foster Family can get things done faster than traditional GUI apps.
+Foster Family is optimised for use via a **Command Line Interface (CLI)**. This means that you primarily interact with it by typing commands. It also retains the benefits of a Graphical User Interface (GUI), allowing you to interact with the application through graphical components. If you can type fast, Foster Family can get things done faster than traditional GUI apps.
 
-This user guide contains **all** the commands available to you, along with **step-by-step explanations** and **examples** to help you master the use of Foster Family. If you are a _new user_, simply head over to [Quick Start](#quick-start) to begin your journey with us. For those who are _already acquainted_, you can refer to the [Table of Contents](#table-of-contents) below to navigate to your command of interest.
+This document is a comprehensive guide to all the commands available to you, along with **step-by-step explanations** and **examples** to help you master the features Foster Family has to offer. If you are a _new user_, we recommend the [Quick Start](#quick-start) to begin your journey. For those who are _already acquainted_, you can refer to the [Table of Contents](#table-of-contents) below to navigate to a section that interests you.
```

Comment:
how about "If your are a new user, please begin your journey with Quick Start (link)", "the Quick Start" sounds a bit weird to me, wdyt

Reply:
I ended up going with "we recommend beginning your journey with Quick Start". Thanks!